### PR TITLE
Initial import

### PR DESCRIPTION
**Pull request by rgreinho**
Signed-off-by: Rémy Greinhofer <remy.greinhofer@gmail.com>

### CODE REVIEW

**Review by mihaimaruseac — COMMENTED** [review]
Thanks.

2 questions:
- should we switch to the new logo?
- do we need `themes/juice/content/...` content?

File: templates/index.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  <h1 class="heading-text">Graph for Understanding Artifact Composition</h1>
+  <h3 class="title-text">
+    aggregates software security metadata into a high fidelity graph database
+  </h3>
+  <p>
+    <a class="github-button" href="https://github.com/guacsec/guac" data-size="large"
+      data-show-count="true" aria-label="Star guacsec/guac on GitHub">Star</a>
+  </p>
+</section>
+<img class="hero-image" style="width: 35%" src="{{ get_url(path="guac.png") }}">
```
[the remark this answers is not in the document]
Comment (rgreinho) [reply templates/index.html:15]: I will! I will also adjust the content, but I first wanted to have some sort of scaffolding running. After that I can make small PRs for the next adjustements.

**Comment by rgreinho** [comment]
>     * do we need `themes/juice/content/...` content?

It does not hurt and gives some examples, so I would leave it there. If it is really a concern, we could totally remove it though since we're not using it as a sub-module.

**Review by mihaimaruseac — APPROVED** [review]

**Comment by mihaimaruseac** [comment]
Let's get this is in then.

Thank you

**Pull request merged**